core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -364,10 +385,12 @@ var StoryEditorPage = function() {
   this.changeNodeOutline = async function(richTextInstructions) {
```

Comment:
Above in `expectChapterExplorationIdToBe`, can we wait for `explorationIdInput` to load before getting its attribute?

Reply:
this still needs to be done